content/events/community_autumn_24/index.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "[Upcoming Event] Meeting ESP komunity - podzimní edice 2024"
+date: 2024-08-28
+draft: false
+description: "Espressif Community Meeting - Autumn 24"
+tags: ["event", "announcement", "community meeting", "czech transcript only"]
+showDate : false
+---
+
+{{< badge >}}
+New event!
+{{< /badge >}}
+
+## Nový Event se blíží
+
+Máme skvělou zprávu! Po úspěšných dvou ročnících tě zveme na naše třetí setkání československého bastlířského eventu, který se letos koná na novém místě – v areálu Vlněna, jen kousek od naší české kanceláře. Jestli tě baví bastlení, programování, zkrátka cokoli kolem našich ESPéček, nebo si prostě zvědavý na naše novinky, tohle si nesmíš nechat ujít!
+
+Těšit se můžeš na zajímavé přednášky, praktické dema, workshop, kde si můžeš vyzkoušet programování v ESP-IDF, a hlavně na skvělou atmosféru mezi lidmi, kteří stejně jako ty milují technologie a objevování nových možností. Přijď si užít večer plný inspirace a setkat se s dalšími nadšenci!
+
+
+### Na co se tedy můžeš těšit:
+
+<article class="gallery">
+  <img src="gallery/RAD_0032.jpg" />
+  <img src="gallery/RAD_0051.jpg" />
+  <img src="gallery/RAD_0060.jpg" />
+  <img src="gallery/RAD_0073.jpg" />
+  <img src="gallery/RAD_0078.jpg" />
+  <img src="gallery/RAD_0503.jpg" />
+  <img src="gallery/RAD_0526.jpg" />
+  <img src="gallery/RAD_0542.jpg" />
+</article>
+
+
+- **Přednášky** Čekají tě inspirativní přednášky od vývojářů z Espressif, lokálních firem a zapálených nadšenců do ESP! Přijď si poslechnout nejnovější trendy, tipy a triky přímo od lidí, kteří tvoří budoucnost těchto technologií.
+
+- **Workshop** Připravili jsme pro tebe dvouhodinový praktický workshop, kde si budeš moci osahat vývoj na našem novém čipu **ESP32C6**. Budeš mít jedinečnou příležitost ponořit se do vývoje s podporou odborníků! Nezapomeň si vzít **vlastní notebook** a pokud máš, přibal si s sebou kabel USB-A - USB-C.
+
+- **Praktické dema** Sestrojili jsme funkční ukázky nových čipů, které nejsou jen na obdivování, ale můžeš si je rovnou sám vyzkoušet! Přijď objevit co všechno ESP dokážou.
+
+- **Networking** Uvolněná atmosféra u pivka s lidmi, kteří sdílejí stejné nadšení! Poznej nové přátele, popovídej si s tech influencery a dej řeč s našimi vývojáři. Toto je ideální příležitost navázat nové kontakty a sdílet své nápady s komunitou.
+
+### Důležité informace na konec:
+
+- **Kdy:** 18. 10. 2024 v 17:00
+- **Kde:** Akce proběhne v areálu Vlněna na adrese **Přízova 3** (vstup je z vnitrobloku Vlněny)
+
+Nepropásni šanci a zaregistruj se co nejdříve ve formuláři níže! Místa jsou omezená a tak si zajisti své místo co nejdříve. **Vstup je zdarma**, ale registruj se prosím jen tehdy, pokud se opravdu chystáš přijít.

```

Comment:
Místa jsou omezená, zajisti si to své co nejdříve... ... **Vstup je zdarma**, ale vyžaduje registraci. Tu prosím vyplň jen v případě, pokud se ...